client/dataEntry.js

```diff
@@ -139,6 +139,24 @@ function getSolveWarnings(resultId, solveIndex) {
   return warnings;
 }
 
+function updateWarningContents () {
```

Comment:
(nit: We don't put spaces between a function name and the parens defining it's arguments.)